lib/applications/schema.ts

```diff
@@ -404,6 +407,7 @@ export default gql`
 
   type CreateNewApplicationResult {
     ok: Boolean!
+    applicationId: Int
```

Comment:
Should this be required?
```suggestion
    applicationId: Int!
```

Reply:
I don't think so? It's not for the other create application results which I assume is because if the application fails to be created, we would return `{ok: false}`, but i don't actually see us doing this anywhere 🥴. There's some todos in the application resolvers for error handling so i presume this would be implemented then, I'll leave it as is for now

Reply:
That's true, good point